config/routes.rb

```diff
@@ -26,6 +26,13 @@
         end
       end
       resources :room_settings, only: %i[update], param: :friendly_id
+      resources :shared_accesses, only: :destroy, path: '/shared_accesses/room', param: :friendly_id do
```

Comment:
I would drop the `path: ...` here

Reply:
I find it annoying that the default path is `/shared_accesses/friendly_id ` when friendly_id is a param of room.